models/stg_stripe__credit_note.sql

```diff
@@ -26,15 +26,14 @@ fields as (
 ),
 
 final as (
-    
     select 
         id as credit_note_id,
-        amount as credit_note_amount,
+        {{ convert_values('amount') }},
```

Comment:
Given the previous comment, can we also update these to include the package namespace.

```suggestion
        {{ stripe_source.convert_values('amount') }},
```

Reply:
Apply the same update for all macro call instances.

Reply:
done!